<launch>
  <arg name="robot" default="scitos_coverage"/>
  <arg name="input_file" default="$(find viper)/data/op/$(arg robot)/aloof/views.json"/>
  <arg name="output_file_values" default="$(find viper)/data/op/$(arg robot)/aloof/view_values.json"/>
  <arg name="output_file_costs" default="$(find viper)/data/op/$(arg robot)/aloof/view_costs.json"/>
  <arg name="output_file_view_keys" default="$(find viper)/data/op/$(arg robot)/aloof/view_keys.json"/>

  <include file="$(find viper)/launch/$(arg robot)/_evalaute_views.launch"/>

  <node name="evaluate_views" pkg="viper" type="eval_views.py" output="screen">
    <param name="robot" type="string" value="$(arg robot)"/>
    <param name="input_file" type="string" value="$(arg input_file)"/>
    <param name="output_file_values" type="string" value="$(arg output_file_values)"/>
    <param name="output_file_costs"  type="string" value="$(arg output_file_costs)"/>
    <param name="output_file_view_keys" type="string" value="$(arg output_file_view_keys)"/>
  </node>
</launch>
